packages/contracts-bedrock/contracts/L1/SystemConfig.sol

```diff
@@ -70,6 +74,19 @@ contract SystemConfig is OwnableUpgradeable, Semver {
      */
     ResourceMetering.ResourceConfig internal _resourceConfig;
 
+    /**
+     * @notice The `signerSet` is a set of addresses that are allowed to issue positive attestations
+     *         for alternative output proposals in the `AttestationDisputeGame`.
+     */
+    address[] internal _signerSet;
```

Comment:
nit: I'd prefer to call them attesters because its possible we end up having multiple types of "signers"